src/credentials/KeytarCredentialManager.ts

```diff
@@ -82,7 +82,7 @@ export = class KeytarCredentialManager extends AbstractCredentialManager {
      */
     public async initialize(): Promise<void> {
         try {
-            this.keytar = await import("keytar");
+            this.keytar = (await import("@zowe/secrets-for-zowe-sdk")).keyring;
         } catch (error) {
             this.loadError = new ImperativeError({
                 msg: "Keytar not Installed",
```

Comment:
When we fail to import keyring from the Secrets SDK, should we change the old error message stating "Keytar not Installed"?